distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/ImagingTest.java

```diff
@@ -0,0 +1,201 @@
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+import org.osgi.service.cm.Configuration;
+
+import ddf.common.test.BeforeExam;
+import ddf.test.itests.AbstractIntegrationTest;
+
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+public class ImagingTest extends AbstractIntegrationTest {
+    private static final String[] REQUIRED_APPS =
+            {"catalog-app", "solr-app", "spatial-app", "imaging-app"};
+
+    private static final String TEST_IMAGE_NITF = "i_3001a.ntf";
+
+    private static final String TEST_MTI_NITF = "gmti-test.ntf";
+
+    @Override
+    protected Option[] configureDistribution() {
+        return options(karafDistributionConfiguration(maven().groupId(
+                "org.codice.alliance.distribution")
+                .artifactId("alliance")
+                .type("zip")
+                .versionAsInProject()
+                .getURL(), "alliance", KARAF_VERSION).unpackDirectory(new File("target/exam"))
+                .useDeployFolder(false));
+    }
+
+    @Override
+    protected Option[] configureCustom() {
+        return options(wrappedBundle(mavenBundle("ddf.test.itests", "test-itests-ddf").classifier(
+                "tests")
+                .versionAsInProject()).bundleSymbolicName("test-itests-ddf")
+                .exports("ddf.test.itests.*"), keepRuntimeFolder());
+    }
+
+    @BeforeExam
+    public void beforeAllianceTest() throws Exception {
+        try {
+            basePort = getBasePort();
+            setLogLevels();
+            getServiceManager().waitForRequiredApps(REQUIRED_APPS);
+            getServiceManager().waitForAllBundles();
+            getCatalogBundle().waitForCatalogProvider();
+            configureSecurityStsClient();
+        } catch (Exception e) {
+            LOGGER.error("Failed in @BeforeExam: ", e);
+            fail("Failed in @BeforeExam: " + e.getMessage());
+        }
+    }
+
+    private String ingestNitfFile(String fileName) throws Exception {
+        InputStream inputStream = getClass().getClassLoader().getResourceAsStream(fileName);
+        byte[] fileBytes = IOUtils.toByteArray(inputStream);
+
+        return given().multiPart("file", fileName, fileBytes, MediaType.APPLICATION_OCTET_STREAM)
+                .expect()
+                .statusCode(HttpStatus.SC_CREATED)
+                .when()
+                .post(REST_PATH.getUrl())
+                .getHeader("id");
+    }
+
+    @Test
+    public void testValidImageNitfMetacard() throws Exception {
+        String id = ingestNitfFile(TEST_IMAGE_NITF);
+
+        String url = REST_PATH.getUrl() + id;
+
+        when().get(url)
+                .then()
+                .assertThat()
+                .body(hasXPath("/metacard[@id='" + id + "']", containsString("isr.image")));
```

Comment:
it's typically better to assert on an `hasXPath` rather than a `containsString`.  I would also assert the mime type is correct as well.